src/client/pages/HLI/HLIStadspas.tsx

```diff
@@ -233,3 +241,61 @@ export default function HLIStadspas() {
     </DetailPage>
   );
 }
+
+function BlockPassButton({ blockPassURL }: { blockPassURL: string }) {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  return (
+    <>
+      <Button
+        variant="secondary"
+        onClick={() => {
+          setIsModalOpen(true);
+        }}
+      >
+        Blokeer deze Stadspas
+      </Button>
+      <Modal
+        title="Weet u zeker dat u uw stadspas wilt blokkeren ?"
+        isOpen={isModalOpen}
+        showCloseButton={false}
+        actions={
+          <ActionGroup>
+            <Button
+              type="submit"
+              variant="primary"
+              onClick={() => {
+                setIsModalOpen(false);
+                if (!blockPassURL) {
+                  return;
+                }
+                fetch(blockPassURL, { method: 'POST' });
```

Comment:
Als het blokkeren fout gaat, wat doen we dan? In dat geval moeten we denk ik een foutmelding tonen in de Modal. 

Wanneer het blokkeren slaagt moet de frontend ge-update worden met de "Is geblokkeerd" functionaliteit maar daar ben je denk ik nog niet aan begonnen. Ik heb wel functionaliteit gemaakt om dit makkelijker te kunnen bewerkstelligen maar is heel lelijk. Voor het blokkeren + updaten van FE kun je het volgende wel gebruiken. https://swr.vercel.app/#overview Ik wil deze package gaan gebruiken in de FE voor soortgelijke functionaliteit.